src/lib/template-instance.ts

```diff
@@ -68,6 +68,7 @@ export class TemplateInstance {
     const parts = this.template.parts;
     let partIndex = 0;
     let nodeIndex = 0;
+    let part: TemplatePart;
```

Comment:
Why hoist this declaration? It's not used outside the loop.

Reply:
Because nested template elements can advance the current part.

Reply:
👍